src/core/api/coap_api.cpp

```diff
@@ -110,6 +110,21 @@ otError otCoapMessageAppendUriPathOptions(otMessage *aMessage, const char *aUriP
     return static_cast<Coap::Message *>(aMessage)->AppendUriPathOptions(aUriPath);
 }
 
+uint16_t otCoapBlockSizeFromExponent(otCoapBlockSize aSize)
+{
+    return static_cast<uint16_t>(1 << (aSize + 4));
```

Comment:
nit: magic number

Reply:
Where is the appropriate place to define these constants? Are defines or enums preferable? Are there any existing examples that you can point me to?

Reply:
enums are generally preferred over defines for scoping purposes. You can find an example here: https://github.com/openthread/openthread/blob/a84fea58e10beaa0d8cb775915c1ec57e221c0a1/src/core/thread/mle_tlvs.hpp#L1740-L1745